update-solution-version/action.yml

```diff
@@ -0,0 +1,21 @@
+# Copyright (c) Microsoft Corporation.
+# Licensed under the MIT License.
+name: 'update-solution-version'
+description: 'Power Platform Update Solution Version'
+inputs:
+  solution-patch-version:
```

Comment:
we should drop the "solution-" prefix for action input names, it's redundant in the action context